### ISSUE TITLE
Fractionnal zoom missing for mouse 

### ISSUE THREAD

**Issue by Walleza1** [openlayers/openlayers#9299]
Default MouseWheel Interaction always zoom by 1 level. 

The [documentation]( https://openlayers.org/en/latest/apidoc/module-ol_interaction.html#.defaults) actually mentions :
> zoomDelta - number - Zoom level delta when using keyboard or mousewheel zoom.

But after taking look at the `ol/interaction.js`, we don't use zoomDelta in constructor : 

```javascript
 const mouseWheelZoom = options.mouseWheelZoom !== undefined ?
    options.mouseWheelZoom : true;
  if (mouseWheelZoom) {
    interactions.push(new MouseWheelZoom({
      condition: options.onFocusOnly ? focus : undefined,
      constrainResolution: options.constrainResolution,
      duration: options.zoomDuration
    }));
}
```

As mentionned here #8423, i can't really change zoomFactor.

**Comment by fredj** [comment]
yes, the doc is wrong: the zoomDelta property is not used for the MouseWheelZoom interaction.

**Comment by Walleza1** [comment]
Is there any plans to add Fractional Zoom feature to MouseWheel Interaction ?
